shared/templates/mount_option/oval.template

```diff
@@ -2,31 +2,61 @@
   <definition class="compliance" id="{{{ _RULE_ID }}}" version="2">
     {{{ oval_metadata(MOUNTPOINT ~ " should be mounted with mount option " ~ MOUNTOPTION ~ ".") }}}
     <criteria operator="OR">
-      <criterion comment="{{{ MOUNTOPTION }}} on {{{ MOUNTPOINT }}}"
-        test_ref="test_{{{ POINTID }}}_partition_{{{ MOUNTOPTIONID }}}_optional_{{{ MOUNT_HAS_TO_EXIST }}}"/>
+      <criteria operator="AND">
+         <criterion comment="{{{ MOUNTOPTION }}} on {{{ MOUNTPOINT }}}"
+           test_ref="test_{{{ POINTID }}}_partition_{{{ MOUNTOPTIONID }}}_optional_{{{ MOUNT_HAS_TO_EXIST }}}"/>
+         <criterion comment="{{{ MOUNTOPTION }}} on {{{ MOUNTPOINT }}} in /etc/fstab"
+           test_ref="test_{{{ POINTID }}}_partition_{{{ MOUNTOPTIONID }}}_optional_{{{ MOUNT_HAS_TO_EXIST }}}_fstab"/>
```

Comment:
Can you explain how this works? I think that when you join the criterion defined by textfilecontent54_test and the criterion defined by partition_test by AND operator they will have to be both true to produce a true result. Bu,t what will happen on the systems where the partition test doesn't work?

Reply:
It will trigger remediation that would do the right thing with the /etc/fstab.